src/components/schedule-visualizer/robot.tsx

```diff
@@ -57,7 +58,12 @@ const Robot = React.forwardRef(function(
             rotate(${-(robot.location.yaw * 180) / Math.PI})`}
           >
             <filter id={`${robot.name}-shadow`} x="-20%" y="-20%" width="140%" height="140%">
-              <feDropShadow dx="0" dy="0" stdDeviation={footprint * 0.15} floodColor="black" />
+              <feDropShadow
+                dx="0"
+                dy="0"
+                stdDeviation={footprint * 0.15}
+                floodColor={inConflict ? 'red' : 'black'}
```

Comment:
Can you use `theme.palette.error.main` (same color of the trajectory) instead of 'red'?